src/System.Net.WebClient/tests/System.Net.WebClient.Tests.csproj

```diff
@@ -30,6 +30,11 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <!-- TODO: Remove these reference once packages are updated -->
```

Comment:
I suspect this is the only one that is actually needed. but I'm fine having the others temporarily as well.